### PR TITLE
Adding Contributing guide and issue and PR templates

### PR DESCRIPTION
**Pull request by Golodhros**
Adding templates for issues and PR
Adding contributing guide based on Britechart's

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+1. Write a failing test for the feature you want to add
+1. On a new terminal, run ``yarn run test:watch``
+1. Check that the test fail
+1. Write the code that would make that test pass on the chart located in ``/src/charts/*/*Component.js`` or ``/src/charts/*/*Chart.js``. Please follow the code style in the current files.
+1. Make the test pass by writing the less amount of code
+1. Refactor until you get a nice and clean code
+1. Check the demo to see the code in action
+1. Update the documentation so that the proper demos gets generated when you run ``yarn run build:storybook``
+1. Create a pull request and ask people of the team to review it
+1. Once you have the OK, merge it!
```
Comment (jchen-eb) [inline CONTRIBUTING.md]: Are the numbers correct here? They are all `1.`

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+1. Given that you new chart is ``ChartName``, create in ``/src/charts/ChartName`` a new file ``ChartName.js`` and test file ``ChartName.test.js``.
+1. Create also a new component in ``/src/charts/ChartName`` a new file ``ChartNameComponent.js`` and test file ``ChartNameComponent.test.js``.
+1. Add the new demo entry on ``/src/stories/index.js``, keeping the same format.
+7. Go to ``webpack.config.js`` and add the new chart to the ``CHARTS`` object, follow the same style.
+8. Go to your test file ``ChartName.test.js`` and, taking another test as an example, copy over pieces until you have the first test case.
+1. Add one test and make it fail callin ``yarn run test:watch``.
+1. Keep on coding according to the code style you can see in the current charts, using the stackedAreaChart as a reference.
+1. Generate your docs with ``yarn run build:storybook`` and manually test the demos.
+1. Create a pull request with your branch and ping one of the core authors to get it reviewed.
+1. Once you have a shipit, merge it
```
Comment (jchen-eb) [inline CONTRIBUTING.md]: Same with these numbers. Oh wait maybe Markdown will turn them into the correct ones?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+4- `git rebase master` & reconcile all conflicts
+5- `git push origin <your branch>`
+6- make your pr with a link to the original issue filed (if you see "unable to merge", please pull from your upstream and rebase again)
+7- be patient :)
+
+---
+
+## Reporting an Issue/Feature Requests
+
+Please use the [issues](https://github.com/eventbrite/britecharts-react/issues) section of the Britecharts-react github profile to submit any bugs found. Please be descriptive and feel free to add pictures or links to docs to make your point as clear as possible.
```
Comment (jchen-eb) [inline CONTRIBUTING.md]: Nit: Britecharts-React

**Comment by jchen-eb** [comment]
LGTM just some minor comments

File: CONTRIBUTING.md (hunk 2 above)
Comment (Golodhros) [reply CONTRIBUTING.md]: Yep, Markdown will do it!

**Pull request merged**